tests/test_crs/valid_fiod_hostpath.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   namespace: my-ripsaw
 spec:
   elasticsearch:
-    server: ES_SERVER
-    port: ES_PORT
+    server: "snafu:passwd@ec2-54-201-210-248.us-west-2.compute.amazonaws.com"
+    port: 9200
```

Comment:
These two keys shouldn't be modified. CI replaces these values at https://github.com/cloud-bulldozer/benchmark-operator/blob/master/test.sh#L68-L69